.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - node
+  - 8.0.0
+sudo: false
+install:
+  - npm install
```

Comment:
Could we use `npm ci` instead of `npm i`?

Reply:
`npm ci` isn't available with npm what comes with Node8

Reply:
Anyways fixed by updating npm before the install command